types/office-js-preview/index.d.ts

```diff
@@ -14561,6 +14573,111 @@ declare namespace Office {
          */
         setAsync(data: string, callback?: (asyncResult: Office.AsyncResult<void>) => void): void;
     }
+    /**
+     * Provides details about the mouse pointer position and the messages or file attachments being dragged and dropped into an add-in's task pane when the
+     * `Office.EventType.ItemDraggedAndDropped` event is raised.
+     *
+     * @remarks
+     * To learn more about the drag-and-drop feature and how to implement it across various Outlook clients, see
+     * {@link https://learn.microsoft.com/office/dev/add-ins/outlook/drag-drop-items | Drag and drop messages and attachments into the task pane of an Outlook add-in}.
+     */
+    export interface DragAndDropEventArgs {
+        /**
+         * Gets the details about the mouse pointer position within an add-in's task pane and the messages or file attachments being dragged and dropped into the task pane.
+         */
+        dragAndDropEventData: DragoverEventData | DropEventData;
+        /**
+         * Gets the type of the event. For details, see {@link https://learn.microsoft.com/javascript/api/office/office.eventtype | Office.EventType}.
+         */
+        type: "olkDragAndDropEvent";
+    }
+    /**
+     * Represents the `DragAndDropEventArgs.dragAndDropEventData` object when messages or file attachments are dragged over an add-in's task pane.
+     *
+     * @remarks
+     * To learn more about the drag-and-drop feature and how to implement it across various Outlook clients, see
+     * {@link https://learn.microsoft.com/office/dev/add-ins/outlook/drag-drop-items | Drag and drop messages and attachments into the task pane of an Outlook add-in}.
+     */
+    interface DragoverEventData {
+        /**
+         * Gets the x-coordinate of the mouse pointer that represents the horizontal position in pixels.
+         */
+        pageX: number;
```

Comment:
Do we need to say that the position is relative to? E.g., is it relative to the task panes upper left corner? To the Outlook window's upper left corner? The device screen's corner?

Reply:
Good question. I'll check with the engineer.